app-modules/portal/src/Http/Controllers/KnowledgeManagementPortal/KnowledgeManagementPortalArticleController.php

```diff
@@ -75,6 +75,11 @@ public function show(KnowledgeBaseCategory $category, KnowledgeBaseItem $article
                 'slug' => $category->slug,
                 'name' => $category->name,
                 'description' => $category->description,
+                'parentCategory' => $category?->parentCategory ? KnowledgeBaseCategoryData::from([
+                    'slug' => $category?->parentCategory->slug,
+                    'name' => $category?->parentCategory->name,
+                    'description' => $category?->parentCategory->description,
```

Comment:
I don't think you should have to null safe `$category` here as there is not a scenario in this Controller that it would be `null`.